src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/scatter_update/scatter_nd_update_kernel_ref.cpp

```diff
@@ -14,17 +14,15 @@ ParamsKey ScatterNDUpdateKernelRef::GetSupportedKey() const {
     k.EnableInputDataType(Datatype::F16);
     k.EnableInputDataType(Datatype::F32);
     k.EnableInputDataType(Datatype::INT32);
+    k.EnableInputDataType(Datatype::INT8);
+    k.EnableInputDataType(Datatype::UINT8);
     k.EnableOutputDataType(Datatype::F16);
     k.EnableOutputDataType(Datatype::F32);
     k.EnableOutputDataType(Datatype::INT32);
     k.EnableOutputDataType(Datatype::INT8);
     k.EnableOutputDataType(Datatype::UINT8);
-    k.EnableInputLayout(DataLayout::bfyx);
-    k.EnableOutputLayout(DataLayout::bfyx);
-    k.EnableInputLayout(DataLayout::bfzyx);
-    k.EnableOutputLayout(DataLayout::bfzyx);
-    k.EnableInputLayout(DataLayout::bfwzyx);
-    k.EnableOutputLayout(DataLayout::bfwzyx);
+    k.EnableAllInputLayout();
+    k.EnableAllOutputLayout();
```

Comment:
As I remember, some format was not supported. Could you specify actually verified formats only?

Reply:
The formats that were not supported, are the formats that are not in the kernel_selector_helper.cpp data_layout list. As far as I know, the kernel_selector_helper.cpp file is a general file that not only applies to scatter_nd_update, but all ops. I could list out and specify only the verified formats only as you asked me (which are the formats in to_data_layout( ) function), but my only concern is if it will be efficient to do so?